frontend/src/components/housing/enquiry/EnquiryForm.vue

```diff
@@ -108,6 +125,15 @@ const onAssigneeInput = async (e: IInputEvent) => {
   }
 };
 
+const showUserLinkModelCheck = (values: Enquiry) => {
+  if (relatedAtsNumber || values.atsClientNumber) return true;
```

Comment:
reason why assigning false and not returning it? could we just have `return (relatedAtsNumber || values.atsClientNumber)`?

Reply:
Simplified.